guides/micronaut-object-storage-aws/metadata.json

```diff
@@ -7,11 +7,10 @@
   "categories": ["AWS"],
   "publicationDate": "2022-10-01",
   "languages": ["java"],
-  "testFramework": "spock",
   "apps": [
     {
       "name": "default",
-      "features": ["object-storage-aws", "graalvm"]
+      "features": ["object-storage-aws", "graalvm", "localstack"]
```

Comment:
I purposely didn't add the `localstack` feature explicitly because then it shows up in the guide:

![image](https://user-images.githubusercontent.com/153880/190976106-84374615-1e44-49c9-951f-21d7b2f3931d.png)

Reply:
I think I can fix that a separate issue. Not including local features in docs.